### ISSUE TITLE
How to use Native Loader without extracting subdirectory

### ISSUE THREAD

**Issue by Jack-Indaboks** [soywod/himalaya#302]
I'm sure this is a dumb question, but here goes:

I have stopped using plugin managers for vim since 8.0 when native plugin loading was added. I usually just git clone the plugin directly into ~/.vim/pack/plugins/start. This is the first plugin I have wanted to use that doesn't appear to have its own git repository. Instead it appears to be a subdirectory of the entire Himalaya source repository. I would rather not clone the whole repo just for the vim plugin. I installed the Himalaya binary from the Arch repo so I don't already have the source repo on my system. I can clone the repo and do a sparse-checkout, but the fact that the plugin is a subdirectory appears to conflict with the vim native plugin format: ```~/.vim/pack/*/(start|opt)/<plugin>```. To get the plugin to load I have to take the vim folder out of the cloned folder (or create a symlink in the right location). Otherwise vim doesn't seem to find it. I'm sure I am missing something simple...

TL;DR: Is there a way to install the vim plugin with the vim 8 native loader and using git for updates, without cloning the entire repo and extracting (or symlinking) a subdirectory?

**Comment by soywod** [comment]
I'm afraid you cannot associate git update and not cloning all himalaya, you will have to make a choice. But you know, the himalaya repo is that big. I mean, the vim folder is sth like 20% (in size) of all the repository itself. The easier for you would be to clone the full repo and do a symlink. You could also add the main himalaya dir to your vim runtimepath (it is what I was doing locally to load my local development vim folder into vim, see https://github.com/soywod/dotfiles/blob/c2c4f0fa4271132b77d1bfd8e4afbfaa6aa6efd9/neovim/config/init.lua#L33)

**Comment by Jack-Indaboks** [comment]
Thank you for your prompt and clear response. I had just settled on doing exactly what you suggested when I stumbled across another package in the Arch repos called "vim-himalaya". It references this repo as upstream (as opposed to any of its predecessors) and was last updated today. It's not git straight to my plugins folder, but it should still work quite well. I think I'll try that first. I love your work so far and look forward to seeing where this project goes. Thank you.
